packages/stark-ui/src/modules/session-ui/session-ui.module.ts

```diff
@@ -16,22 +15,41 @@ import { translationsFr } from "./assets/translations/fr";
 import { translationsNl } from "./assets/translations/nl";
 import { mergeUiTranslations } from "../../common/translations";
 
+import { STARK_SESSION_UI_CONFIG, StarkSessionUiConfig } from "./entities";
+import { StarkSessionTimeoutWarningDialogComponent } from "./components/session-timeout-warning-dialog.component";
+import { MatDialogModule } from "@angular/material/dialog";
+import { StarkLoggingModule, StarkSessionModule, StarkLocale } from "@nationalbankbelgium/stark-core";
+import { EffectsModule } from "@ngrx/effects";
+import { StarkSessionTimeoutWarningDialogEffects } from "./effects";
+
 @NgModule({
 	declarations: [
 		StarkLoginPageComponent,
 		StarkPreloadingPageComponent,
 		StarkSessionExpiredPageComponent,
-		StarkSessionLogoutPageComponent
+		StarkSessionLogoutPageComponent,
+		StarkSessionTimeoutWarningDialogComponent
+	],
+	exports: [
+		StarkLoginPageComponent,
+		StarkPreloadingPageComponent,
+		StarkSessionExpiredPageComponent,
+		StarkSessionLogoutPageComponent,
+		StarkSessionTimeoutWarningDialogComponent
 	],
-	exports: [StarkLoginPageComponent, StarkPreloadingPageComponent, StarkSessionExpiredPageComponent, StarkSessionLogoutPageComponent],
 	imports: [
 		CommonModule,
 		UIRouterModule.forChild({
 			states: SESSION_UI_STATES
 		}),
+		StarkSessionModule,
+		StarkLoggingModule,
```

Comment:
Please remove these two modules otherwise we are creating a hard link between the different modules of Stark.

Our goal is to have soft links, meaning that we just require the services (which the developer can provide by his own) but not the Stark modules themselves.